### DISCUSSION TITLE
Chip component with inputContainer

### DISCUSSION THREAD

**Discussion by FelixBlaisThon** [mantinedev/mantine#6313] [Q&A] [answered]
Why some input components have `inputContainer` and others don't? Let's say I want to render a tooltip over the `Chip` component, it would be much simpler with the inputContainer

**Comment by rtivital** [comment, marked as answer]
https://mantine.dev/core/chip/#chip-with-tooltip

**Reply by FelixBlaisThon** [reply]
woah yikes. My bad

**Reply by FelixBlaisThon** [reply]
I was thinking, what is the thought process of this vs adding an inputContainer prop ?
